.vsts-pr.yml

```diff
@@ -37,7 +37,7 @@ stages:
     parameters:
       pool:
         name: $(DncEngPublicBuildPool)
-        demands: ImageOverride -equals 1es-windows-2022-open
+        demands: ImageOverride -equals windows.vs2022preview.scout.amd64.open
         os: windows
       helixTargetQueue: windows.amd64.vs2022.pre.open
```

Comment:
I think it's supposed to be this one and remove the .pre

Reply:
but I know they have both 17.11 and 17.10 so not sure which is 17.10 specifically

Reply:
No.

Reply:
Sorry, the Helix queue only applies to the Helix tests. The tests for TemplateEngine run on the build machine since they are not Helix tests. They are just normal unit tests.